prowler/providers/aws/services/ec2/ec2_service.py

```diff
@@ -269,16 +269,28 @@ def __describe_network_interfaces__(self, regional_client):
                     #         'GroupName': 'default',
                     #     },
                     # ],
-                    for sg in interface.get("Groups", []):
-                        for security_group in self.security_groups:
-                            if security_group.id == sg["GroupId"]:
-                                security_group.network_interfaces.append(eni)
+                    self.__add_network_interfaces_to_security_groups__(
```

Comment:
Maybe my point was not clear, I thought that this should be called in the `init` function after calling `__describe_network_interfaces__` to let that function to do their thing as we do in all the service's functions.

Reply:
I did it like that so we don't have to iterate again through the ENIs.